tests/integration/009_team/main.tf

```diff
@@ -22,4 +22,24 @@ output "team_resource_description" {
 
 output "team_data_description" {
   value = data.env0_team.team_data.description
-}
\ No newline at end of file
+}
+
+resource "env0_custom_role" "custom_role1" {
+  name = "custom-role-${random_string.random.result}"
+  permissions = [
+    "CREATE_PROJECT"
+  ]
+}
+
+resource "env0_custom_role" "custom_role2" {
+  name = "custom-role-${random_string.random.result}2"
```

Comment:
if it's random-string, why do you need `2` at the end? it will be different than name from custom-role1, no?

Reply:
it generates one random string - that way their names vary.